src/components/common/Modal/index.js

```diff
@@ -0,0 +1,104 @@
+import React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+import { AiOutlineClose } from "react-icons/ai";
+
+import Portal from "./Portal";
+import theme from "../../../config/constants/theme";
+
+export default function Modal({ children, isOpen, setIsOpen }) {
+  if (!isOpen) {
+    return null;
+  }
+
+  const handleOnOuterClick = (e) => {
```

Comment:
저희 컨벤션 맞추려면 여기도 줄이지 않고 써야하지 않을까 의견드립니다!

```suggestion
  const handleOnOuterClick = (event) => {
```

Reply:
좋은 의견입니다! 컨벤션에 맞춰서 풀네임으로 수정하겠습니다

Reply:
`handleOuterClick`이 좋은 것 같습니다!